4.classify-features/classify-features.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Classify Cell Health Nuclei Features
+# 
+# ### Import libraries
+
+# In[1]:
+
+
+import pandas as pd
+import pathlib
+import numpy as np
+import urllib.request
+import joblib
```

Comment:
Consider removing unused imports.

```suggestion
import pathlib
import urllib.request
import joblib
```